### ISSUE TITLE
If the Context Menu is already open, and you again right click, then it opens system context menu

### ISSUE THREAD

**Issue by ankitsalu** [msarsha/ng2-right-click-menu#78]

**Comment by msarsha** [comment]
True, it's the default behavior and I see no reason to prevent it from opening.
Would love to discuss if you have another opinion 🙂

**Comment by ankitsalu** [comment]
Thanks for replying back, i need the exact behavior shown in  demo project i.e. https://msarsha.github.io/ng2-right-click-menu/
It always shows the custom context menu everytime i right click which is what i want, but right now when i right click it shows me custom menu but when i again click then original system menu opens up....so its like to menu at same time....I want the exact behavior shown in  demo project....but i am confused how to achieve this?

**Comment by msarsha** [comment]
Oh, I understand, I though you are right clicking the custom menu.
I will look into implementing this behavior.

**Comment by ankitsalu** [comment]
> Oh, I understand, I though you are right clicking the custom menu.
> I will look into implementing this behavior.

But how is it done on - demo project? .....i think this already feature already exists and working but i am not getting which property to set to enable this feature.... https://msarsha.github.io/ng2-right-click-menu/

**Comment by msarsha** [comment]
The implementation is completely different on the current version (I should also update the demo project) so there is no way to enable it atm.

**Comment by ankitsalu** [comment]
> The implementation is completely different on the current version (I should also update the demo project) so there is no way to enable it atm.

thanks :-) but are you planning do to this? i need to implement this solution, and not have much time...many thanks for your help

**Comment by msarsha** [comment]
Hopefully I will have some free time this evening to check it out.
You are also more then welcome to try implementing it and submit a PR - this will be much appreciated.
😄

**Comment by ankitsalu** [comment]
> Hopefully I will have some free time this evening to check it out.
> You are also more then welcome to try implementing it and submit a PR - this will be much appreciated.
> 😄

thanks a lot :-) i will definitely try 👍
